src/Raven.Server/Documents/Handlers/QueriesHandler.cs

```diff
@@ -147,6 +149,7 @@ private async Task Query(QueryOperationContext queryContext, OperationCancelToke
             try
             {
                 result = await Database.QueryRunner.ExecuteQuery(indexQuery, queryContext, existingResultEtag, token).ConfigureAwait(false);
+                AddQueryTimingsToTrafficWatch(indexQuery);
```

Comment:
can we add it after writing the response? probably after 'AddPagingPerformanceHint'